feat(registry): add support to artifact registry

joaomlopes opened the pull request:
Bump gcp-cli and gcp-gcr Orbs to support the extra new parameter `repository` when wanting to push the image to GCP Artifact Registry instead of Container Registry

david-montano-circleci commented:
Hello @joaomlopes . This looks good. I just want to check if we can run tests for this so let me see what I can do.
Thank you for your contributions!

joaomlopes commented:
> Hello @joaomlopes . This looks good. I just want to check if we can run tests for this so let me see what I can do. Thank you for your contributions!

Of course! Let me know if you need anything else from me on this! I appreciate you taking the time!

david-montano-circleci commented:
@joaomlopes I fixed the pipeline of this orb. Can you make a small commit to see if the pipelines can correctly report the CircleCI result back?
I already tested this and it looks ok.

joaomlopes commented:
> @joaomlopes I fixed the pipeline of this orb. Can you make a small commit to see if the pipelines can correctly report the CircleCI result back? I already tested this and it looks ok.

@david-montano-circleci Definitely! I just removed an extra line on one of the files! Thank you!

joaomlopes commented:
Hi @david-montano-circleci . Is there something here that I can do to help?

david-montano-circleci commented:
Hello @joaomlopes . This PR is failling in the current pipelines. But it does not seem to be tied to the PR but to ieither ts dependencies or the old executor it uses. I am still looking into it in this branch: https://github.com/CircleCI-Public/gcp-gke-orb/tree/pr/75

david-montano-circleci commented:
@joaomlopes please update this PR with this change: https://github.com/CircleCI-Public/gcp-gke-orb/commit/532fb418fca14e3bd62a4b148b1ad80e0be40634
That should fix the pipeline errors
